airbrakes/state.py

```diff
@@ -113,18 +112,21 @@ def update(self):
 
         data = self.context.data_processor
 
-        # If our current speed is less than our max speed, that means we have stopped accelerating
+        # If our current velocity is less than our max velocity, that means we have stopped accelerating
         # This is the same thing as checking if our accel sign has flipped
-        # We make sure that it is not just a temporary fluctuation by checking if the speed is a bit less than the max
-        # speed
-        if data.speed < data.max_speed - data.max_speed * MAX_SPEED_THRESHOLD:
+        # We make sure that it is not just a temporary fluctuation by checking if the velocity is a
+        # bit less than the max velocity
+        if data.vertical_velocity < data.max_vertical_velocity - data.max_vertical_velocity * MAX_VELOCITY_THRESHOLD:
             self.next_state()
             return
 
         # Fallback: if our motor has burned for longer than its burn time, go to the next state
-        if time.time() - self.start_time > MOTOR_BURN_TIME:
-            self.next_state()
-            return
+
+        # =======DISABLED==========
+        # if time.time() - self.start_time > MOTOR_BURN_TIME:
+        #     print('fallback switch')
+        #     self.next_state()
+        #     return
```

Comment:
Do we want to enable this back? I think we should, since we do need a fallback condition

Reply:
We should, but we need to figure out what our motor burn time is, and add maybe a quarter second to compensate for the velocity to reach 0.97*maxVelocity

Reply:
It will be much safer to have the motor_burn_time constant be a higher number rather than lower. Switching too early could lead to apogee prediction converging 3 or 4 seconds later, or worse, not converging at all. If we switch too late, we might lose a half-second or second of coast phase, but it will still converge quickly.

Reply:
ok. But if we are going with that, I'd like us to use the timestamp from the data packets, and not system time, since we don't want to include cpu run time.